doc/reference/devices_usb.md

```diff
@@ -1,22 +1,33 @@
 (devices-usb)=
 # Type: `usb`
 
+```{youtube} https://www.youtube.com/watch?v=SAord28VS4g
+```
+
 ```{note}
 The `usb` device type is supported for both containers and VMs.
 It supports hotplugging for both containers and VMs.
 ```
 
 USB devices make the specified USB device appear in the instance.
+For performance issues, avoid using devices that require high throughput or low latency.
+
+For containers, only `libusb` devices (at `/dev/bus/usb`) are passed to the instance.
+This method works for devices that have user-space drivers.
+For devices that require dedicated kernel drivers, use a [`unix-char` device](devices-unix-char) or a [`unix-hotplug` device](devices-unix-hotplug) instead.
```

Comment:
Is that correct as an explanation?
Not 100% I understood it correctly.

Reply:
I think that makes sense.